src/main/features/claimant-response/draft/courtDecision.ts

```diff
@@ -2,5 +2,6 @@ export enum DecisionType {
   CLAIMANT_IN_FAVOUR_OF_DEFENDANT = 'CLAIMANT_IN_FAVOUR_OF_DEFENDANT',
   CLAIMANT = 'CLAIMANT',
   DEFENDANT = 'DEFENDANT',
-  COURT = 'COURT'
+  COURT = 'COURT',
```

Comment:
another one of these... why do we have 3 of them?

Reply:
Yes we should have common area for Decision type, as for some reason we have duplicates within code base